app/src/main/java/com/infomaniak/mail/utils/LoginUtils.kt

```diff
@@ -123,7 +123,7 @@ class LoginUtils @Inject constructor(
         infomaniakLogin.deleteToken(
             okHttpClient = HttpClient.okHttpClientNoTokenInterceptor,
             token = apiToken,
-            onError = { SentryLog.e("DeleteTokenError", "API response error: $it") },
+            onError = { SentryLog.i("DeleteTokenError", "API response error: $it") },
```

Comment:
See with @LunarX for this one.

Reply:
Moved to https://github.com/Infomaniak/android-kMail/pull/2037